### PR TITLE
Dpp 1363 fix integration tests

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

**Comment by scottywakefield** [comment]
@Pauljgraham these integration tests are much more like acceptance tests. I'd suggest they don't serve much purpose and aren't worth maintaining. I'll be removing all existing integration tests from one of the components I'm currently working on.

**Comment by Pauljgraham** [comment]
They currently aren't being used. The Only value I see in it, is it may help us understand some of the data tables in the solution and how they relate to the individual processes.

**Review by scottywakefield — APPROVED** [review]

**Comment by satya-sanikommu** [comment]
@Pauljgraham @scottywakefield Looks like this PR has not been merged, is this still required ??

@scottywakefield You cherry picked some integration changes from a branch and merged into Develop. Are these the same changes ???

**Comment by scottywakefield** [comment]
@satya-sanikommu the cherry picked changes were for DPP-1361.

**Comment by satya-sanikommu** [comment]
@scottywakefield Thanks, yes just realised that. Also changed the BaseBranch to develop for these changes to go into, as TransferBaseBranch_2 has been merged into Develop and has been deleted.

File: src/PeriodEnd/LevyPayments/SFA.DAS.ProviderPayments.Calc.LevyPayments.IntegrationTests/Utilities/TestDataHelper.cs [hunk 1]
```diff
@@ -106,18 +106,34 @@ internal static decimal[] GetAccountBalance(long id)
                 deliveryYear = Query<int>("SELECT Calendar_Year FROM LevyPayments.vw_CollectionPeriods WHERE Collection_Open = 1").Single();
             }
 
-            Execute("INSERT INTO PaymentsDue.RequiredPayments "
+            // TODO Review this. I added field names and guessed the values based on the column order. However I had to comment out 3 fields and
```
Comment (scottywakefield) [inline src/PeriodEnd/LevyPayments/SFA.DAS.ProviderPayments.Calc.LevyPayments.IntegrationTests/Utilities/TestDataHelper.cs]: so we still need to sort out this comment @Pauljgraham.

**Review by scottywakefield — CHANGES_REQUESTED** [review]

**Review by scottywakefield — APPROVED** [review]

**Review by carlbm — APPROVED** [review]
All tests green now

**Comment by satya-sanikommu** [comment]
Review and Testing completed. Ready to merge

**Pull request merged**